chore: update sleep.ts example

janeklb opened the pull request:
## Motivation

The jsdoc `@example` snippet is probably leftover from v2 since it's using `yield` instead of `yield*`. Same goes for the [scope](https://frontside.com/effection/docs/scope) and [streams and subscriptions](https://frontside.com/effection/docs/collections) (for `subscription.next()`) docs.

cowboyd reviewed (approved):
@janeklb Thanks for taking the time to correct this!

janeklb commented:
My pleasure! Awesome library - really appreciate all your thoughts (posts) around structured concurrency.

ps. Had to push again as I hadn't been signing my commits

janeklb committed:
chore: update sleep.ts example

The jsdoc `@example` snippet is probably leftover from v2 since it's using `yield` instead of `yield*`

janeklb committed:
update docs

cowboyd commented:
The fixes for the guides are live. The type docs are versioned with the release of the code, and so will appear with the next release. Again, thanks!
